packages/ui-components/src/components/select-option/select-option.tsx

```diff
@@ -41,17 +54,18 @@ export class KvSelectOption implements ISelectOption, ISelectOptionEvents {
 						'disabled': this.disabled,
```

Comment:
I'm having a look at this so it doesn't break the previous bottom slot projection

Reply:
I've exposed a css prop `--select-option-text-flex-direction` that allows for the direction of the text-container to be styled to use either a flex row or column layout.